backend/controllers/prisonerProfile/prisonerCsraReview.js

```diff
@@ -0,0 +1,78 @@
+const { putLastNameFirst, formatName, formatTimestampToDate } = require('../../utils')
+const { notEnteredMessage } = require('../../common-messages')
+
+const csraTranslations = {
+  LOW: 'Low',
+  STANDARD: 'Standard',
+  HI: 'High',
+}
+
+module.exports = ({ prisonApi }) => async (req, res, next) => {
+  const { offenderNo } = req.params
+  const { assessmentSeq } = req.query
+
+  try {
+    const prisonerDetails = await prisonApi.getDetails(res.locals, offenderNo)
+    const { bookingId, firstName, lastName } = prisonerDetails
```

Comment:
I don't think this works for reviews from a previous location. I believe the bookingId needs to come from the link - it is part of the details returned for each CSRA in the history page. BTW you can test with prisoner `G2458GG` on T3